src/components/form/InputGroup.vue

```diff
@@ -0,0 +1,14 @@
+<template>
+  <div class="form-group">
+    <label>{{ title }}</label>
+    <slot></slot>
+  </div>
+</template>
+<script>
+import t from "vue-types";
+export default {
+  props: {
+    title: t.String,
```

Comment:
Does this work?  
I believe it should be: 
```suggestion
    title: t.string,
```